core/app/models/spree/promotion_rule_user.rb

```diff
@@ -0,0 +1,8 @@
+module Spree
+  class PromotionRuleUser < Spree::Base
+    self.table_name = 'spree_promotion_rules_users'
```

Comment:
I remember adding this in the beginning because I didn't want to make too many specs break all at once as well as thinking about deprecations.

Now that you have it switched over, the next step is to probably remove the new `.table_name` and set it back to Rails convention.

`.table_name` is usually used for legacy data, except in this situation, we have control to rename it.